src/condor_tests/cmd_q_protocols.run

```diff
@@ -207,6 +210,7 @@ my $secondappend_condor_config = '
 	ALL_DEBUG = D_FULLDEBUG
 	NUM_CPUS = 1
 	CONDOR_Q_USE_V3_PROTOCOL = false
+	# SHADOW_LAZY_QUEUE_UPDATE = false
```

Comment:
As far as it goes, I wanted to leave it in there but maybe should have added a comment about why: because turning off SHADOW_LAZY_QUEUE_UPDATE breaks the test, which means it way less robust than it at first may appear.

Reply:
A comment would be useful, but not required for this ticket.